screenpy/actions/silently.py

```diff
@@ -45,17 +47,19 @@ def perform_as(self, actor: Actor) -> None:
 
     def __init__(self, duck: Performable):
         if not isinstance(duck, Performable):
-            raise NotPerformable(
-                "SilentlyPerformable only works with Performable. "
-                "Use `Silently` instead."
+            msg = (
+                "SilentlyPerformable only works with Performables."
```

Comment:
The corresponding test needs updating with this change.

Reply:
There was no change to the message, i only moved the string into a `msg` variable.

Reply:
Notice that all the tests on the PR pass.

... though the lint stages are broken, because i removed `pylint`, `isort`, etc.

Reply:
... the eggface is my face, from the egg on it. My face, that is.

You're talking about the `s`. 🤦